substrate/client/network/src/service.rs

```diff
@@ -1732,15 +1691,6 @@ where
 					};
 					let reason = match cause {
 						Some(ConnectionError::IO(_)) => "transport-error",
-						Some(ConnectionError::Handler(Either::Left(Either::Left(
-							Either::Left(Either::Right(
-								NotifsHandlerError::SyncNotificationsClogged,
-							)),
-						)))) => "sync-notifications-clogged",
-						Some(ConnectionError::Handler(Either::Left(Either::Left(
-							Either::Right(Either::Left(_)),
-						)))) => "ping-timeout",
```

Comment:
It looks like we don't propagate the `ping-timeout` and `sync-clogged` labels anymore, are they merged into `actively-closed`?

Reply:
I suspect they were merged into `Io`, `ConnectionError::Handler` variant doesn't exist upstream anymore